pysys/__init__.py

```diff
@@ -46,8 +46,8 @@
 
 """
 
-__author__  = "Moray Grieve"
-"""The original author of PySys."""
+__author__  = "Moray Grieve, Ben Spiller"
+"""The current and previous lead maintainers of PySys."""
```

Comment:
Should that perhaps be "original and current lead maintainers" considering the order of the names?

Reply:
good idea, fixed